### PR TITLE
[Peter Latona] - Sage - Creating a Smart Contract for Decentralized Voting

### PR DESCRIPTION
**Pull request by SasaniEldis**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: src/data-tutorials/users.tsx [hunk 1]
```diff
@@ -789,6 +789,13 @@ const Users: User[] = [
     website: 'blog/tutorials/React-Native-&-Celo-Easily-build-React-Native-dApps-on-Celo',
     tags: ['intermediate','foundation', 'reactnative', 'composer'],
   },
+  {
+    title: 'Creating a Smart Contract for Decentralized Voting',
+    description: 'learn how to build a decentralised voting application using smart contracts on the Celo blockchain',
+    preview: require('./showcase/intermediate/Creating-a-Smart-Contract-for-Decentralized-Voting.png'),
```
Comment (Julius170) [inline src/data-tutorials/users.tsx:795]: The name of the image reference on preview has the be the exact same name of the image you uploaded

File: blog/12-05-23-Creating-decent-sc-voting/creating-sc-voting.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+title: Creating a Smart Contract for Decentralized Voting
+description: In this tutorial, you’ll learn how to build a decentralised voting application using smart contracts on the Celo blockchain.
+authors:
+  - name: Peter Latona
+    title: contributor, Celo sages
+    url: https://github.com/SasaniEldis
+    image_url: https://avatars.githubusercontent.com/u/107583957?v=4
+tags: ["intermediate", "celo", "celosage", "smartcontract"]
+hide_table_of_contents: true
+slug: /tutorials/creating-a-Smart-Contract-for-Decentralized-Voting
```
Comment (Julius170) [inline blog/12-05-23-Creating-decent-sc-voting/creating-sc-voting.md:11]: This has to be the exact same with "website" in your user.tsx file

File: blog/12-05-23-Creating-decent-sc-voting/creating-sc-voting.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+- Learning about other use cases for smart contracts such as decentralised finance (DeFi) or non-fungible tokens (NFTs).
+
+## About the Author
+
+Imole Peter L.
+
+A web3 enthusiast, content writer for web3 brands, visual artist, and seasoned author (Pen name:[Sasani Eldis](https://www.amazon.com/Imole-Latona/e/B088F4KF7H)). Connect with me on[LinkedIn](https://www.linkedin.com/in/imole-peter-latona)
+
+## References
+
```
Comment (Julius170) [inline blog/12-05-23-Creating-decent-sc-voting/creating-sc-voting.md:142]: can you also provide the Github repo link for this  tutorial

File: src/data-tutorials/users.tsx (hunk 1 above)
Comment (Julius170) [reply src/data-tutorials/users.tsx:795]: It's not the name of the image you uploaded in the intermediate folder

**Review by Julius170 — COMMENTED** [review]
Hi @SasaniEldis your piece looks good but You'll need to check all your slug details and make sure they are correct and well synced to the appropriate file

**Review by Julius170 — COMMENTED** [review]
Hi @SasaniEldis your piece looks good but You'll need to check all your slug details and make sure they are correct and well synced to the appropriate file

File: blog/12-05-23-Creating-decent-sc-voting/creating-sc-voting.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+authors:
+  - name: Peter Latona
+    title: contributor, Celo sages
+    url: https://github.com/SasaniEldis
+    image_url: https://avatars.githubusercontent.com/u/107583957?v=4
+tags: ["celosage", "smartcontract", "intermediate"]
+hide_table_of_contents: true
+slug: /tutorials/creating-a-Smart-Contract-for-Decentralized-Voting
+---
+
```
Comment (Julius170) [inline blog/12-05-23-Creating-decent-sc-voting/creating-sc-voting.md:13]: add your header image on this line

**Review by Julius170 — COMMENTED** [review]
just one more change 🥂

**Review by Julius170 — COMMENTED** [review]
just one more change 🥂

**Comment by viral-sangani** [comment]
Closing Celo Sage PRs

**Pull request closed without merging**